packages/opentelemetry-plugin-fetch/src/fetch.ts

```diff
@@ -174,6 +174,9 @@ export class FetchPlugin extends core.BasePlugin<Promise<Response>> {
   ): void {
     let resources: PerformanceResourceTiming[] = resourcesObserver.entries;
     if (!resources.length) {
+      if (!performance.getEntriesByType) {
```

Comment:
Please add unit test for that - use `sandbox`

Reply:
Done, wanted to do a cleaner solution but ran into a bunch of issues